test/parallel/test-fs-rm.js

```diff
@@ -130,6 +130,17 @@ function removeAsync(dir) {
   }));
 }
 
+// Removing a .git directory should not throw an EPERM.
+// Refs: https://github.com/isaacs/rimraf/issues/21.
+{
+  const gitDirectory = nextDirPath();
+  fs.mkdirSync(gitDirectory);
+  execSync(`git -C ${gitDirectory} init`);
```

Comment:
I don't think we should assume git is present, e.g. you could be testing from a source tarball.

Reply:
Could we perhaps make the test dependent on a `.git` folder existing in the current folder?

Reply:
I agree that tests should not fail if `git` is not available – skipping this test would be fine by me, ignoring the `Command not found` error would work too.

Reply:
Updated the test to check for the presence of git before running the tests, PTAL.